install/install.sh

```
@@ -0,0 +1,184 @@
+            fi
+        done < "$requirements_file"
+    else
+        echo "Error: $requirements_file not found."
+        return 1
+    fi
+}
+
+repo_install(){
+	clean_dir "$repo"
+	clone_repo $INSTALL_VER $repo
+	cd loveletter
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)